### ISSUE TITLE
Missing Value Error

### ISSUE THREAD

**Issue by jefferh** [jesusmmp/exams2sakai#1]
I'm getting a "missing value" error when I run the example from the documentation. Here's the example code: 

```
library("exams2sakai")
options(device.ask.default = FALSE)
myexam <- list( "boxplots", "regression", "scatterplot", "relfreq" )
dir.create(mydir <- tempfile())
exams2sakai( myexam, n = 1, verbose = TRUE, dir = "mydir", points = c( 1 ) )
```

Here's the error I'm getting:

```
Error in if (typemchoice == TRUE || typemchoice == T) { : 
  missing value where TRUE/FALSE needed
```

Here's some of my system info:

```
R version 4.3.3 (2024-02-29)
Platform: aarch64-apple-darwin20 (64-bit)
Running under: macOS Sonoma 14.3.1
```

**Comment by jesusmmp** [comment]
Hi @jefferh!
Let us take a look. 

Thanks!

**Comment by jesusmmp** [comment]
Hi @jefferh!

There is a problem of compatibility of our package with the new version of exams 2.4-0.
Please use 2.3.6 until we fix it. Thanks!

Apologies.

**Comment by jesusmmp** [comment]
Hi @jefferh!

I've just released a new version (v0.4) which should fix this issue.
Please could you test it?

Thanks!

**Comment by jesusmmp** [comment]
This issue has been resolved on the new release v0.4
